aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -9,16 +9,21 @@ module aptos_framework::aggregator_v2 {
     /// The value of aggregator underflows (goes below zero). Raised by uncoditional sub() call
     const EAGGREGATOR_UNDERFLOW: u64 = 2;
 
-    /// Element type of the aggregator not supported. Raised by create_aggregator() call,
-    /// if Element type is not supported (currently u64 or u128).
-    const EAGGREGATOR_ELEMENT_TYPE_NOT_SUPPORTED: u64 = 3;
-
     /// The generic type supplied to the aggregator snapshot is not supported.
     const EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE: u64 = 5;
```

Comment:
Why 5?

Reply:
I'm using 3 and 4 in the validate_history function when an overflow/underflow is raised in the original execution, but should not have been raised when the speculative_start_value is replaced with a new value.

Reply:
Why validate_history would have a different error? An overflow is an overflow I no matter where it comes from I think or maybe I am missing something